doc/ProvisionReleaseBuild.md

```diff
@@ -13,30 +13,30 @@
    
    - Install Provisioner API and requisite packages
    ```yum install -y yum-utils
-      yum-config-manager --add-repo "${CORTX_RELEASE_REPO}/3rd_party/"
-      yum-config-manager --add-repo "${CORTX_RELEASE_REPO}/cortx_iso/"
-
-      cat <<EOF >/etc/pip.conf
-      [global]
-      timeout: 60
-      index-url: $CORTX_RELEASE_REPO/python_deps/
-      trusted-host: $(echo $CORTX_RELEASE_REPO | awk -F '/' '{print $3}')
-      EOF
+   yum-config-manager --add-repo "${CORTX_RELEASE_REPO}/3rd_party/"
```

Comment:
Apparently, this is wrong.

This change is not needed.
But, we need a fix for the Line 15.
"yum install -y yum-utils" should be on a new line.